content/doc/developer/handling-requests/json.adoc

```diff
@@ -0,0 +1,255 @@
+---
+title: Expose HTTP API with json content with Jenkins
+summary: How to write GET and POST HTTP API with Jenkins, and how to test it.
+layout: developersection
+references:
+- url: https://github.com/stapler/stapler/blob/master/docs/reference.adoc
+  title: Stapler URL Binding Reference
+- url: ../routing
+  title: How requests in Jenkins are routed
+- url: ../../testing
+  title: Testing Jenkins
+---
+
+== Expose HTTP API with json content with Jenkins
+
+It's a common requirement for automation to expose Json object over HTTP API in server. This page show how a plugin can create a Jenkins Extension to expose HTTP API for GET and POST request, and how to test it with JenkinsRules (from acceptance-test-harness).
```

Comment:
The first sentence is a little cryptic and will shy unexperienced people away. Worth trying to rephrase and give some short use cases.

Reply:
I change it to summarize the content of the page instead.